lib/phoenix_component.ex

```diff
@@ -1789,6 +1810,8 @@ defmodule Phoenix.Component do
     """
   )
 
+  attr.(:autocomplete, :boolean, default: false)
```

Comment:
List it in global include instead?

Reply:
Let's also change `<.link ...>` to use this feature.

Reply:
Yes will undo my recent changes and use this :)